### ISSUE TITLE
Use <?php instead of <?

### ISSUE THREAD

**Issue by faquin** [squix78/espaper-server-php#4]
https://github.com/squix78/espaper-server-php/blob/4c17a35e05bce54888894ea9827a6c793d369d36/ESPaperCanvas.php#L120

On my php server, I need to write <?php instead of <? if not the tag is not recognised.
Same apply for the line 121

**Comment by lorol** [comment]
Yes I already mention that in #2 :) but basically it is your (and my) problem. 
However it is good to have a consistent format for all php tags.
